dialogue-clients/src/main/java/com/palantir/dialogue/clients/ReloadingClientFactory.java

```diff
@@ -137,6 +148,111 @@ default boolean dnsNodeDiscovery() {
         Optional<ExecutorService> blockingExecutor();
     }
 
+    private static final class ErrorSerializationFormatSettingConjureRuntime implements ConjureRuntime {
+        private final ConjureRuntime delegate;
+        private final ConjureErrorParameterFormat errorParameterSerializationFormat;
+
+        ErrorSerializationFormatSettingConjureRuntime(
+                ConjureRuntime delegate, ConjureErrorParameterFormat errorParameterSerializationFormat) {
+            this.delegate = delegate;
+            this.errorParameterSerializationFormat = errorParameterSerializationFormat;
+        }
+
+        @Override
+        public BodySerDe bodySerDe() {
+            return new ErrorParameterSerializationFormatSettingBodySerDe(
```

Comment:
do we want to make this a private member variable?

Reply:
Makes sense. Updated.